Codecademy/gamut
# test(SelectDropdown): unskips and fixes tests

### Overview


Unskips SelectDropdown tests and fixes them


### PR Checklist

- [x] Related to JIRA ticket: [GM-955]

#### Testing Instructions


Confirm the tests pass




[GM-955]: https://skillsoftdev.atlassian.net/browse/GM-955?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
